src/cpu/cpu_convolution_list.cpp

```diff
@@ -138,12 +138,13 @@ const std::map<pk_dt_impl_key_t, std::vector<impl_list_item_t>> &impl_list_map()
             CPU_INSTANCE_AARCH64_ACL(acl_wino_convolution_fwd_t)
             CPU_INSTANCE_AARCH64(brgemm_1x1_convolution_fwd_t<sve_512>)
             CPU_INSTANCE_AARCH64(brgemm_convolution_fwd_t<sve_512>)
-            CPU_INSTANCE_AARCH64(jit_sve_512_dw_convolution_fwd_t)
+            CPU_INSTANCE_AARCH64(jit_uni_dw_convolution_fwd_t<sve_512,data_type::f32>)
             CPU_INSTANCE_AARCH64(jit_sve_512_1x1_convolution_fwd_f32_t)
             CPU_INSTANCE_AARCH64(jit_sve_convolution_fwd_t<f32,f32,f32,sve_512>)
             CPU_INSTANCE_AARCH64_ACL(acl_depthwise_convolution_fwd_t)
             CPU_INSTANCE_AARCH64_ACL(acl_indirect_gemm_convolution_fwd_t)
             CPU_INSTANCE_AARCH64_ACL(acl_gemm_convolution_fwd_t<f32>)
+            CPU_INSTANCE_AARCH64(jit_uni_dw_convolution_fwd_t<sve_256,data_type::f32>)
```

Comment:
Could you move this above ACL in the list please?

I've done some benchmarking at the oneDNN and PyTorch levels, and it looks like for Neoverse-V1, this implementation is better than the existing ACL implementation. This is really great work, thank you!

Reply:
Hi @jondea , will check and do the needful

Reply:
Hi @jondea, thank you for reviewing my PR. I have done the needful changes.